src/components/MobilePage/MobilePage.tsx

```diff
@@ -71,9 +70,6 @@ export default class MobilePage extends React.PureComponent<Props, State> {
             <div className="success">{t('mobile_page.success')}</div>
           )}
         </Form>
-        <span className="suggestion">
-          Participate in our contest for a chance to <a href="http://contest.decentraland.org">earn MANA and LAND</a>
```

Comment:
Use:
```
{t('mobile_page.subtitle_link.before')}
<a href="${REACT_APP_CONTEST_SITE}">{t('mobile_page.subtitle_link.text')}</a>
{t('mobile_page.subtitle_link.after')}
```
with:
```
"mobile_page": {
  ...
  "subtitle_link": {
    "before": "Participate in our contest for a chance to",
    "text": "earn MANA and LAND",
    "after": "."
  },
  ...
}
```

Reply:
I think the text is not longer used

Reply:
Yeah, that bit is removed—RFD inappropriate to give a looping CTA on an error